### ISSUE TITLE
Docs issue: Setup is a bit confusing

### ISSUE THREAD

**Issue by Frijol** [greenkeeperio/greenkeeper#493]
(Possibly this issue belongs on the Website repo, you tell me)

Having some challenges getting started, hopefully this information is helpful for you!

1. Difficulty installing a Github app: it wasn't obvious to me how to install as a github app. I first installed on my personal account to test it out, then went to enable on my organization. But the app wasn't installed on my organization! The account page for my organization told me to go to the app configuration settings on Github (https://github.com/organizations/tessel/settings/installations/), but there was no Greenkeeper app there, which was confusing. Eventually I went looking in my personal account settings, found Greenkeeper installed in the parallel settings location. Still couldn't figure out how to get it installed on the org (tried uninstalling from my personal account to see if a new button would become available, tried the Your Account section on greenkeeper.io). Eventually clicked around on the website and re-triggered the installation process to get Greenkeeper installed on my organization.
2. Everything is red! In my account/the org repos, all of the repos are highlighted red, as in not enabled. Debug information at the bottom of the account page is useful, but doesn't appear to fit my situation (I didn't close anything, there is a package.json and CI on the repo). So, looks like Greenkeeper is having a problem which causes there to be no initial PR. In fact, there is no `greenkeeper/initial` branch. My suspicion is that I have installed but not enabled greenkeeper (docs say: "Important: just having the Greenkeeper installed on a repo doesn’t necessarily enable it yet. More on that below. By the way, you can change this behaviour later on the Installed integrations page in your organizations’ settings.") but I can't find the promised information on installing vs. enabling, it sounds like if there is a package.json a PR will be sent.
3. Ah! Found: "Important: If all dependencies are already up-to-date, we currently won’t send this initial pull request. Instead, Greenkeeper will enable itself on the repo immediately, and you’ll start getting new issues on this repo only when Greenkeeper determines that there’s something for you to do. You can control this by only installing the app on the repos you actually need it on, in step 1 above. Again, we highly recommend taking the time to whitelist repos individually." So, maybe it's working, it just doesn't look like it? Is there any way to tell, as a new user, the difference between a repo that needs to be reset and a repo that is doing great and needs no updates?

In conclusion, I think maybe I have it set up just fine? Hard to tell, and I'm happy to help with a docs PR if you can confirm!

**Comment by Frijol** [comment]
Oh! It's working! New branches just appeared (after I followed suggestion to whitelist per repo). Does it just take a few minutes?

**Comment by rmehner** [comment]
Hey there,

thanks for this report again. Let me try to clarify :)

1. To install Greenkeeper, head to https://github.com/apps/greenkeeper and then hit the "Install" button. That button might read "Configure" if you happen to have the app installed already. In the next screen you can pick the organization and then you need to go to "Installed GitHub Apps" on the left and pick Greenkeeper. From there on you can add and remove repositories to Greenkeeper.

(I'll go through a new user install flow with the team today to check how we can improve docs, just to make sure we give people the right directions)

2. We're working on improving the display for "installed, but not enabled yet" from within the account app. In fact @espy spend a good chunk of time on that. In short: Greenkeeper has three states:

- Not installed at all
- Installed, but not enabled
- Enabled

To enable Greenkeeper, you must merge the initial PR that it sends after it was installed. There are some special cases, where we don't send the PR and just enable you right away (OSS repo? All deps up to date and no major versions upgradable? Badge already added? Then you are good to go and we enabled the repo straight away)

There are also cases where we don't send the initial PR and don't enable the repo, for example when it is a fork or it doesn't have issues enabled. Also if we detect that you don't have a CI installed (by not seeing commit statuses set on the branch commits after a certain timeout), we'll open up an issue to tell you that we need CI to work.

3. Does the badge say "enabled"? If so, all should be good :) Please note we only send PRs / open issues, if either something breaks by a new version that was released, or if the version is outside of your version range specified in the package.json.

Given the following package.json:

```json
{
  "dependencies": {
    "lodash": "^1.0.0"
  }
}
```

Scenario 1: lodash releases `1.0.1`, all CI tests pass => You will not receive a PR or an issue
Scenario 2: lodash releases `1.0.1`, your tests fail => We'll notify you about that breakage by opening up an issue ("An in-range update of lodash is breaking the build") and will also check if pinning lodash to 1.0.0 will fix the build and if so, will notify you in the issue. We'll also keep checking if any new lodash version that is published fixes this issue and will also notify you about that in the issue.
Scenario 3: lodash relases `2.0.0`, we will create a PR with the new version and the PR status will tell you if the build is breaking or not :)

These are the three main scenarios I can think of right away. Hope that helped.

> Oh! It's working! New branches just appeared (after I followed suggestion to whitelist per repo). Does it just take a few minutes?

Depending on queue sizes on our side and your CI's side, plus the time the tests need to run, it can take a while, yes :) Also we only check for updates (after the initial branch) whenever there is a new npm package version published of any of your dependencies.

Hope this makes it a bit clearer. We definitely can and will improve on the documentation side of things. Happy to hear any suggestions you might have.

Once again: Thanks for taking your time for this extensive report, super helpful! 👍

**Comment by espy** [comment]
Hi @Frijol ! Thanks so much for the feedback!

1. I just noticed this too, I‘m currently updating the account dashboard to give concise installation instructions instead of just saying that there aren’t any repos for a user/org, and link to the proper installation page for each, so people don’t run in to this into the future.

2 and 3: I‘m also adding notes that red can also mean that the CI is running for the initial pull request, and that the delay depends on your tests. 

A proper docs section for the website is also in the works.

Sorry you ran in to all these, but thanks a million for telling us :)

**Comment by Frijol** [comment]
Sounds good, glad I could be of help! It is working fine now on my repo– so I think the documentation generally had me perform the right steps, but could be most improved by helping a user understand what to expect (the setting for "enabled but CI hasn't finished running yet" will be particularly helpful there).
Thanks for making Greenkeeper!

**Comment by janl** [comment]
docs fixed :D
